galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -75,6 +75,32 @@ def create(self, request, *args, **kwargs):
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def update(self, request, *args, **kwargs):
+        groups = []
+        for account in request.data['groups']:
+            if account == RH_PE_ACCOUNT_SCOPE:
+                groups.append(account)
+            else:
+                group, _ = auth_models.Group.objects.get_or_create_identity(
+                        auth.RH_ACCOUNT_SCOPE, account.split(':')[1])
```

Comment:
Is it worth passing in `account.split(':')[0]` rather than `auth.RH_ACCOUNT_SCOPE`? If `account` was for some reason `not-rh-identity-account:foo` the database would get a new entry `rh-identity-account:foo`

Reply:
> Is it worth passing in account.split(':')[0] rather than auth.RH_ACCOUNT_SCOPE?

:+1: 

> If account was for some reason `not-rh-identity-account:foo` the database would get a new entry `rh-identity-account:foo

I can update this to use the prefix from the request, rather than `RH_ACCOUNT_SCOPE`. However, it's not possible in the proposed UI to submit group with a different prefix (see [here](https://github.com/ansible/ansible-hub-ui/pull/87/files#diff-19cc5312e1df3810642c295604c5329bR257)).

This comment raises the same concern as @newswangerd did in the UI part of this issue: https://github.com/ansible/ansible-hub-ui/pull/87#discussion_r368721153.

AFAIK, we don't use other types than `rh-identity-account` and `system:partner-engineer`. Do we plan to introduce other?